cmd/root.go

```diff
@@ -28,6 +28,10 @@ import (
 	"github.com/spf13/viper"
 )
 
+const (
+	MB = 1048576 // in bytes
```

Comment:
I've just looked and its astounding that there isn't a standard variable for this.